curvefs/src/metaserver/inode_manager.cpp

```diff
@@ -165,37 +165,171 @@ MetaStatusCode InodeManager::DeleteInode(uint32_t fsId, uint64_t inodeId) {
     return MetaStatusCode::OK;
 }
 
-MetaStatusCode InodeManager::UpdateInode(const Inode &inode) {
-    VLOG(1) << "UpdateInode, " << inode.ShortDebugString();
+MetaStatusCode InodeManager::UpdateInode(const UpdateInodeRequest &request) {
```

Comment:
Why modify the input parameter?

Reply:
Because the previous implementation will copy inode outside the function，and cause a concurrency problem.